AI-and-Analytics/Getting-Started-Samples/IntelPython_XGBoost_GettingStarted/README.md

```diff
@@ -1,18 +1,18 @@
-# `Intel® Python XGBoost* Getting Started` Sample
+# Intel® Python XGBoost* Getting Started Sample
```

Comment:
is this product name correct? AI Tools Selector page and names database show Intel® Optimization for XGBoost*

Reply:
updated